terraform/shared/scripts/setup.sh

```diff
@@ -110,8 +109,24 @@ sudo apt-get install -y docker-ce
 
 # Needs testing, updating and fixing
 if [[ ! -z ${INSTALL_NVIDIA_DOCKER+x} ]]; then 
+
+  # key was rotated, see https://forums.developer.nvidia.com/t/notice-cuda-linux-repository-key-rotation/212771
+  sudo apt-key del 7fa2af80
+  # auto install - FAILS
```

Comment:
Leftover debug code?

Reply:
Indeed. The old key can be removed but it's not necessary, although I'm not sure how much of a security threat it poses when old keys are still being added to apt.